### DISCUSSION TITLE
Slidev (sli.dev)

### DISCUSSION THREAD

**Discussion by snth** [catppuccin/catppuccin#2155] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Hi there,

I'd love a catppuccin theme for sli.dev.

There are a number of community themes listed on the website: https://sli.dev/themes/gallery.html#community-themes

I don't know what's involved but given the number of other community theme contributions I think it would be relatively straightforward (I hope).

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by aurkaxi** [comment]
+1

**Comment by yodatak** [comment]
A unocss port of catppuccin for helping porting it https://github.com/catuhana/unocss-catppuccin

**Reply by aurkaxi** [reply]
how do i use this in slidev?

**Reply by catuhana** [reply]
> how do i use this in slidev?

https://sli.dev/custom/config-unocss#configurations

**Comment by nik-rev** [comment]
I did it!

https://github.com/catppuccin/catppuccin/issues/2614

**Reply by snth** [reply]
Thank you!

**Comment by sgoudham** [comment]
Closing as any further discussion should take place in #2614
